Sources/TuistCore/Models/PackageSettings.swift

```diff
@@ -18,28 +18,39 @@ public struct PackageSettings: Equatable, Codable {
     /// The custom project options for each project generated from a swift package
     public let projectOptions: [String: XcodeGraph.Project.Options]
 
+    /// A Boolean value that indicates whether the test targets of local swift packages are included in generated project.
+    ///
+    /// If only SPM project, test targets are always included regardless of the value of this property.
```

Comment:
Let's copy the documentation changes suggested above

Reply:
It's done.